app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
-[ ] please make sure the belongs_to side between Comment and User is formated the same as in the Post class